### DISCUSSION TITLE
Quarto vs R Markdown?

### DISCUSSION THREAD

**Discussion by turalsadigov** [quarto-dev/quarto-cli#424] [Q&A] [answered]
Hi everyone. 

Can someone explain why there is a change, i.e., why did R Studio not update R Markdown but create a new system, Querto, instead? To me, it just looks like R Markdown with a few changes here and there. Thanks.

**Comment by jjallaire** [comment, marked as answer]
The overall goal of Quarto is to make the process of creating and collaborating on scientific and technical documents dramatically better. The number of languages and runtimes used for scientific discourse is very broad (and the Jupyter ecosystem in particular is extraordinarily popular). Quarto is fundamentally multi-language and multi-engine (supporting Knitr, Jupyter, and Observable today and potentially other engines tomorrow).

On the other hand, R Markdown is fundamentally tied to R which severely limits the number of practitioners it can benefit. You could argue that we can just teach everyone R (and tell them to abandon Jupyter) in order to create reproducible publications, however in practice this ends up being a non-starter.

Quarto is really our attempt to bring R Markdown to everyone. While it is a "new" system, it should also be noted that it is highly compatible with existing content: you can render most R Markdown documents and Juptyer notebooks unmodified with Quarto. The concept is to make a major, long term investment in reproducible research, while keeping it compatible with existing formats and adaptable to the various environments users work in.
